src/Cli/dotnet/NugetPackageDownloader/LocalizableStrings.resx

```diff
@@ -138,4 +138,7 @@
   <data name="FailedToValidatePackageSigning" xml:space="preserve">
     <value>Failed to validate package signing.</value>
   </data>
-</root>
+  <data name="FailedToGetPackageUnderPackageSourceMapping" xml:space="preserve">
+    <value>Package Source Mapping is enabled but no feeds matched the specified packed ID: {0}. See the documentation for Package Source Mapping at https://learn.microsoft.com/nuget/consume-packages/package-source-mapping#enable-by-manually-editing-nugetconfig for more details.</value>
```

Comment:
It would be worth making this an aka.ms link instead. Let me know if you need help doing that.